src/main/java/frc/robot/constants/SHOOTER.java

```diff
@@ -0,0 +1,81 @@
+package frc.robot.constants;
+
+import edu.wpi.first.math.util.Units;
+
+public final class SHOOTER {
+  public static double gearRatioBottom = 1.0;
+  public static double gearRatioTop = 1.0;
+
+  public static final int kSlotIdx = 0;
+
+  public static final double kP = 10;
+  public static final double kI = 0.0;
+  public static final double kD = 1;
+
+  public static final double kS = 0.0;
+  public static final double kV = 0.0;
+  public static final double kA = 0.0;
+
+  public static final double kShooterAngle = Units.degreesToRadians(35);
+
+  public static final double kDistanceFromIntake = Units.inchesToMeters(19);
+
+  public enum WAIT {
+    WAIT_FOR_FLYWHEEL_SETPOINT(3),
+    WAIT_FOR_AMP_SCORE(0.8);
+
+    private final double value;
+
+    WAIT(double value) {
+      this.value = value;
+    }
+
+    public double get() {
+      return value;
+    }
+  }
+
+  public enum FLYWHEEL_STATE {
```

Comment:
Change this to SETPOINT and make the values all in RPM. Update all commands that use this to use the appropriate value